test/backtrace.jl

```diff
@@ -5,10 +5,29 @@ have_backtrace = false
 for l in bt
     lkup = ccall(:jl_lookup_code_address, Any, (Ptr{Void},), l)
     if lkup[1] == :backtrace
-        @test lkup[4] == false # fromC
+        @test lkup[6] == false # fromC
         have_backtrace = true
         break
     end
 end
 
-@test have_backtrace
+macro issue1334_inlinedcode()
+    :(begin error("foo") end)
+end
+
+let inlinedfile, inlinedline
+    function testinline()
+        @issue1334_inlinedcode
```

Comment:
i might suggest adding a few lines to this function to make the backtrace test a little harder. for example:
```julia
function
  1
  @issue1334
  2
end
```